### DISCUSSION TITLE
Multiselect conditions

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15122] [Feature improvement]
WF on multi-select combo field doesn't contain operator "contain"

![image](https://user-images.githubusercontent.com/84110093/131411951-edaef1b3-aaa8-427c-9929-aebab44bf160.png)

Regards

**Comment by bpabiszczak** [comment]
Screenshot from the last version

![image](https://user-images.githubusercontent.com/10197525/131412916-08bdaabd-9666-427d-a303-de922018db5a.png)

**Comment by s2b-git** [comment]
I have upgraded system from 6.1 to latest and this operator dont shows. I added condition to /modules/Vtiger/helpers/AdvancedFilter.php, but conditions on two values in multi-select field are not respected. Is there any other file I should fix?

regards

**Comment by bpabiszczak** [comment]
6.1 https://gitstable.yetiforce.com/index.php?module=Accounts&view=List&mid=51&parent=47

![image](https://user-images.githubusercontent.com/10197525/131414374-eababc26-f442-4973-a12d-63d081167dde.png)
